tether:
  delta: 0.5
  eq_tolerance: 0.00001
  collision_threshold: -0.03 # more negative number means tether gets entangled more easily
  L_max: 8.0
  #nvblox_collision: 0.1

initial_position:
  x: 0.0
  y: 2.0
  z: 3.0


tether_color:
  r: 1.0  # Red
  g: 0.0  # Green
  b: 0.0  # Blue
  a: 1.0  # Alpha (transparency)

direct_path_color:
  r: 1.0  # Red
  g: 0.0  # Green
  b: 0.0  # Blue
  a: 1.0  # Alpha (transparency)

safe_path_color:
  r: 0.0  # Red
  g: 0.0  # Green
  b: 1.0  # Blue
  a: 1.0  # Alpha (transparency)


ropepath_color:
  r: 0.6  # Red
  g: 0.6  # Green
  b: 0.0  # Blue
  a: 1.0  # Alpha (transparency)




rovpath_color:
  r: 0.8  # Red
  g: 0.8  # Green
  b: 0.0  # Blue
  a: 1.0  # Alpha (transparency)
